### PR TITLE
CI

### PR DESCRIPTION
**Pull request by ssolson**
initial CI test commit

Apologizes guys if you are getting a lot of notifications. The only way to get this working is by pushing the changes up.

### CODE REVIEW

File: .github/workflows/python-package.yml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  build:
+
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: ["ubuntu-latest"] #, "macos-latest", "windows-latest"]
+        python-version: [3.7, 3.8, 3.9]
+
+    steps:
```
Comment (H0R5E) [inline .github/workflows/python-package.yml:22]: Sterling,  you are not checking out the repository (it doesn't do it automatically). You need to add:

```
      - name: Set up Git repository
        uses: actions/checkout@v2
```

File: .github/workflows/python-package.yml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+#        shell: bash -l {0}
+#        run: conda info    
+    - name: Set up Git repository
+      uses: actions/checkout@v2
+    - name: Install pip flake8 and pytest
+      run: |
+        python -m pip install --upgrade pip
+        python -m pip install flake8 pytest 
+    - name: Create environment
+      run: |
```
Comment (H0R5E) [inline .github/workflows/python-package.yml:39]: Sterling, I'm watching these builds and I think you need to define a shell before your run command for conda to work properly. So like:

```
- name: Conda info
        shell: bash -l {0}
        run: conda info
```

It's described [here](https://github.com/marketplace/actions/setup-miniconda#example-1-basic-usage).

**Comment by H0R5E** [comment]
Whoop!

**Comment by ssolson** [comment]
@ryancoe this is ready for a final review. This branch is passing for python 3.7, 3.8, 3.9 on WIndows, Mac, and linux.

**Pull request merged**